### PR TITLE
update rate limits for for vector tiles keyless access

### PR DESCRIPTION
**Pull request by rmglennon**
We don't list the keyless limits for vector tiles, and had someone contact us about 429 errors.

From @nvkelso: 
"keylessQpd": 2400,
"keylessQpm": 120,
"keylessQps": 30,

Assuming 15 tiles / map view:
"keylessQpd": 2400, = 160 original map views per minute (unlimited cached)
"keylessQpm": 120, = 8 original map views per minute (unlimited cached)
"keylessQps": 30, = 2 original map views per second (unlimited cached)

This is good info, but I don't know what to do with it because it explains what you'd expect to see in real terms. Maybe a second paragraph on vector tiles, instead of putting it in with the other services?

### CODE REVIEW

**Comment by nvkelso** [comment]
It's possible we will lower keyless vector tile limits in the next 7 days.

**Comment by nvkelso** [comment]
Assigning to me to update with rate limits once that changes.

**Comment by nvkelso** [comment]
Keyless APi limits changed in `prod` this morning, I've updated the docs here.

@kkowalsky to add language about "cache" usage, and general review.

**Comment by kkowalsky** [comment]
~ adding some copy about cached tiles ~

**Comment by nvkelso** [comment]
Related issue about cache documentation is https://github.com/mapzen/mapzen-docs-generator/issues/229.

File: docs/overview/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The services have maximum numbers of queries you can make within a certain perio
 
 All the projects used to build the Mapzen-hosted services are open source. If you want to try Mapzen's products, start with the hosted services to see if they fit your workflow needs. If you later decide that you need additional customizations or higher capacity, you can consider installing on your own servers the open-source code used to build Mapzen's services.
 
-If you send a query without a valid API key (keyless access), the rate limits for Mapzen Search, Turn-by-Turn, Matrix, and Elevation are reduced to 1,000 requests per day, 6 per minute, and 1 per second.
+If you send a query without a valid API key (keyless access), the rate limits for Mapzen Search, Turn-by-Turn, Matrix, Elevation, and Vector Tiles are reduced to 1,000 requests per day, 6 per minute, and 1 per second.
```
Comment (rmglennon) [inline docs/overview/index.md]: remove the service names here if they are all the same now? "If you send a query without a valid API key (keyless access), the rate limits are 1,000 requests per day, 6 per minute, and 1 per second for each Mapzen service."

**Comment by kkowalsky** [comment]
once this passes, I'm going to merge & then make a subsequent PR to solve #229

**Pull request merged**